sources/packages/api/src/route-controllers/institution-locations/institution-location.institutions.controller.ts

```diff
@@ -125,25 +127,10 @@ export class InstitutionLocationInstitutionsController extends BaseController {
   @Patch(":locationId")
   async update(
     @Param("locationId") locationId: number,
-    @Body() payload: InstitutionLocationFormAPIInDTO,
-    @UserToken() userToken: IInstitutionUserToken,
-  ): Promise<void> {
-    // Validate the location data that will be saved to SIMS DB.
-    const dryRunSubmissionResult = await this.formService.dryRunSubmission(
-      FormNames.InstitutionLocation,
+    @Body() payload: InstitutionLocationPrimaryContactAPIInDTO,
```

Comment:
institution user will have the ability to update only `
  primaryContactFirstName,
  primaryContactLastName,
  primaryContactEmail,
  primaryContactPhone`  ?